src/ert/storage/local_ensemble.py

```diff
@@ -338,22 +367,61 @@ def _load_dataset(
         return xr.combine_nested(datasets, "realizations")
 
     def load_parameters(
-        self, group: str, realizations: Union[int, npt.NDArray[np.int_], None] = None
+        self,
+        group: str,
+        realizations: Union[int, Tuple[int], npt.NDArray[np.int_], None] = None,
     ) -> xr.Dataset:
-        return self._load_dataset(group, realizations)
+        try:
```

Comment:
This causes the method `_load_dataset` to become unused so you can just delete it.